lib/board.rb

```diff
@@ -2,18 +2,15 @@
 require_relative 'player'
 class Board
   include UserInterface
-  attr_reader :option
   attr_accessor :state, :player_move, :current_player, :player1, :player2, :message, :cells, :next_player
```

Comment:
[OPTIONAL] Could you please double-check that all of these need to be `attr_accessor` or if you could move some of them into `attr_reader` to have better control of what others (including yourselves from the future) can do with your class ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))